README.md

```diff
@@ -29,7 +29,10 @@ Content is released under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.
 > <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0</a>
 > license.</a>
 
-Code, including source files and code samples if any in the content, is released under [CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/).
+Code, including source files and code samples if any in the content, is released under [CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/), with the following exceptions:
+
+* The primer components in the `node_modules` directory are under the [MIT license](https://github.com/primer/octicons/blob/master/LICENSE)
```

Comment:
I don't know where the canonical "MIT license" lives, so I copied the link from the Primer README :grinning: